model/Hardware/Properties/mass.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# mass
+
+## Summary
+
+information related to physical hardware.
+
+## Description
+
+Mass is defined as an intrinsic property of a body for describing a physical hardware component.
+
+## Metadata
+
+- name: mass
+- Nature: DataProperty
+- Range: xsd:Number
```

Comment:
```suggestion
- Range: xsd:decimal
```

Maybe `xsd:decimal` or something else?

`Number` is not existed in XSD
https://www.w3.org/TR/xmlschema11-2/#built-in-primitive-datatypes